utils/BookstoreApiUtils.js

```diff
@@ -0,0 +1,93 @@
+const { expect, request } = require("@playwright/test");
+const { UtilsFunctions } = require("./utilsFunctions");
+let utilsFunctions = new UtilsFunctions();
+const urlApi = utilsFunctions.parseLocalJson("../fixtures/ApiUrl.json");
+const booksData = utilsFunctions.parseLocalJson("../fixtures/bookstore.json");
+const { DELETED, CREATED } = require('./statusCodes');
+
+
+export class BookstoreApiUtils {
+
+  async getBookIsbnByIndex(index) {
+    const apiContext = await request.newContext();
+    const getBooks = await apiContext.get(urlApi.books, {
+      headers: {
+        "Content-Type": "application/json"
+      },
+    });
+
+    return (await utilsFunctions.parseResText(getBooks)).books[index].isbn;
+
+  }
+
+  async postBooks({
+    isbn = "",
+    userId = "",
+    token = "",
+    statusCode = CREATED,
+    incorrectIsbn = false,
+    wrongUserId = false,
+    noAuth = false
+  }) {
+    const apiContext = await request.newContext();
+    const addBooks = await apiContext.post(urlApi.books, {
```

Comment:
leave method open for testing in all methods. For example try to send delete on this URL, or patch or any other

Reply:
Implementirano u svim postojecim metodama